listenbrainz/webserver/static/js/src/YoutubePlayer.tsx

```diff
@@ -139,10 +142,9 @@ export default class YoutubePlayer
     const { onTrackNotFound } = this.props;
     // We use cueVideoById("") as a means to clear any playlist.
     // If search or loadVideoByID yield no results we can detect that nothing was found
-    if (
-      this.youtubePlayer.getVideoData &&
-      !this.youtubePlayer.getVideoData().video_id
-    ) {
+    const videoData =
+      this.youtubePlayer.getVideoData && this.youtubePlayer.getVideoData();
```

Comment:
I am a bit confused what this does.

Reply:
I tried to adress it and make it more declarative, and improved the comment. Let me know if that's clearer